modules/hardware/x86_64-generic/kernel/host/configs/usb.config

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: CC-BY-SA-4.0
+CONFIG_USB_SUPPORT=y
+CONFIG_USB=y
+CONFIG_USB_STORAGE=y
```

Comment:
We could place `CONFIG_USB_STORAGE` in `debug.config` fragment as well. 
I don't see how it would be needed on host in generic case for `-release`-mode.

Reply:
That case I think all `usb.config` can be moved only for `-debug`-mode assuming release builds be flashed on NVME.